lib/cn/kvset.c

```diff
@@ -522,54 +486,28 @@ kvset_open2(
 
     assert(ks->ks_kvsetid != 0);
 
-    /* map single hblock */
-    err = mpool_mcache_mmap(mp, 1, &km->km_hblk_id, &ks->ks_hmap);
+    err = kvset_hblk_init(mp, km->km_hblk_id, &ks->ks_vgmap, &ks->ks_use_vgmap,
+        &ks->ks_hlog, &ks->ks_hblk);
```

Comment:
Fit on one line?

Reply:
Is this payback? :)

Reply:
Does not fit.

Reply:
Hahaha!  That was not my intent, but I had the same thought after I wrote the comment ;-)

Reply:
100 columns or 80?  I think our limit is 100.

Reply:
💯

Reply:
doesn't fit into 100